src/main/java/seedu/planner/model/ModulePlanner.java

```diff
@@ -49,7 +49,7 @@
     private final ObservableList<Module> takenModules = FXCollections.observableArrayList();
 
 
-    private Map<ProgrammeRequirement, int[]> statusMap = new HashMap<>();
+    private Map<String, Integer> statusMap = new LinkedHashMap<>();
```

Comment:
Why change from `ProgrammeRequirement` (stronger type) into `String` (weaker type)? If it is due to the lack of `"University Level Requirement"` enum, just add it to `ProgrammeRequirement` and use that instead.

Reply:
Done